apps/mobile-wallet/src/features/walletconnect/WalletConnectPairingsModal.tsx

```diff
@@ -0,0 +1,112 @@
+/*
+Copyright 2018 - 2024 The Alephium Authors
+This file is part of the alephium project.
+
+The library is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+The library is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with the library. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+import { useEffect } from 'react'
+import { useTranslation } from 'react-i18next'
+import { Image } from 'react-native'
+import styled from 'styled-components/native'
+
+import AppText from '~/components/AppText'
+import BottomButtons from '~/components/buttons/BottomButtons'
+import Button from '~/components/buttons/Button'
+import EmptyPlaceholder from '~/components/EmptyPlaceholder'
+import { ModalScreenTitle, ScreenSection } from '~/components/layout/Screen'
+import ListItem from '~/components/ListItem'
+import { useWalletConnectContext } from '~/contexts/walletConnect/WalletConnectContext'
+import BottomModal from '~/features/modals/BottomModal'
+import { closeModal } from '~/features/modals/modalActions'
+import { ModalContent } from '~/features/modals/ModalContent'
+import withModal from '~/features/modals/withModal'
+import { useAppDispatch } from '~/hooks/redux'
+
+interface WalletConnectPairingsModalProps {
+  onPasteWcUrlPress: () => void
+  onScanQRCodePress: () => void
+}
+
+const WalletConnectPairingsModal = withModal<WalletConnectPairingsModalProps>(
+  ({ id, onPasteWcUrlPress, onScanQRCodePress }) => {
+    const { t } = useTranslation()
+    const dispatch = useAppDispatch()
+    const { unpairFromDapp, walletConnectClient, activeSessions } = useWalletConnectContext()
+
+    useEffect(() => {
+      if (!walletConnectClient) dispatch(closeModal({ id }))
+    }, [activeSessions.length, dispatch, id, walletConnectClient])
+
+    const handleDisconnectPress = async (pairingTopic: string) => {
+      await unpairFromDapp(pairingTopic)
+    }
+
+    const handlePasteWcUrlPress = () => {
+      onPasteWcUrlPress()
+    }
+
+    const handleScanQRCodePress = () => {
+      onScanQRCodePress()
+    }
```

Comment:
That's a bit redundant wouldn't you say? It's literally a rename, exactly the same as

```ts
    const handlePasteWcUrlPress = onPasteWcUrlPress

    const handleScanQRCodePress = onScanQRCodePress
```

so , personally, I'd remove it alltogether

Reply:
Fixed in d3c078e8b0bfb90b9555a4a41190c727095d6161